stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldUI.kt

```diff
@@ -41,11 +50,21 @@ import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.text.AnnotatedString
 import androidx.compose.ui.text.input.ImeAction
 import androidx.compose.ui.unit.dp
+import com.stripe.android.core.Logger
+import com.stripe.android.uicore.BuildConfig
 import com.stripe.android.uicore.R
 import com.stripe.android.uicore.stripeColors
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.launch
 
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+val LocalAutofillEventReporter = staticCompositionLocalOf<(String) -> Unit> {
+    {
+        Logger.getInstance(BuildConfig.DEBUG)
+            .debug("LocalAutofillEventReporter $it event not reported")
```

Comment:
Do we ever replace this with a real implementation?

Reply:
Yes, we do, see here: https://github.com/stripe/stripe-android/pull/5672/files#diff-1c107527c6f1e85c0fa4a8e0ef3316663596f8448afaea3ea83e1c632a9b4f48R154